src/Tracy/Helpers.php

```diff
@@ -20,7 +20,7 @@ class Helpers
 	 */
 	public static function editorLink($file, $line = NULL)
 	{
-		$file = strtr($origFile = $file, Debugger::$editorMapping);
+		$file = self::editorMapping($origFile = $file);
```

Comment:
Yeah - that's what I was originally planning on doing, but I thought I should follow the approach that @dg took yesterday when he implemented substr_replace.

I am certainly happy to go with the array_keys / array_values approach though - it's certainly less code.

Would you like me to modify the PR?